dockerfiles/theia/Dockerfile

```diff
@@ -20,9 +20,6 @@ ENV GITHUB_TOKEN=$GITHUB_TOKEN
 
 ARG THEIA_GITHUB_REPO=eclipse-theia/theia
 
-# Define upstream version of theia to use
-ARG THEIA_VERSION=master
```

Comment:
as I understood, it has been reworked to come from the `build.include`
https://github.com/eclipse/che-theia/blob/287faa2566b8d7b9da8c335c580d8956054dfe22/build.include#L14
and looks like it's not used anymore

Reply:
@benoitf correct me please if I'm wrong here

Reply:
https://github.com/eclipse/che-theia/blob/master/dockerfiles/theia/docker/alpine/builder-patch-theia.dockerfile#L5-L7 ?